deps/vend.go

```diff
@@ -212,7 +220,7 @@ func Vend(pkgs []string, format string, options ...VendOptions) error {
 		}
 	}
 
-	// if a lock or hold flag is present, or if and update was requested and a
+	// if a lock or hold flag is present, or if an update was requested and a
```

Comment:
Not part of issue #48 addressed in the PR, but simply corrects a typo on a comment.